console/src/view/resources.rs

```diff
@@ -75,12 +84,20 @@ impl TableList for ResourcesTable {
                     let resource = resource.upgrade()?;
                     let resource = resource.borrow();
 
+                    let parent_id_str = match resource.parent_id() {
+                        Some(id) => {
+                            format!("{}", id)
+                        }
+                        None => "n/a".to_string(),
```

Comment:
another string that shouldn't have to be `format!`ed every time we render a frame...